build/spas.ts

```diff
@@ -325,30 +325,49 @@ export async function buildSPAs(options) {
   }
 }
 
-async function fetchRecentContributions() {
+async function fetchGitHubPRs(repo = "mdn/content", count = 5) {
   const twoDaysAgo = new Date(Date.now() - 48 * 60 * 60 * 1000);
   const pullRequestsQuery = [
-    "repo:mdn/content",
+    `repo:${repo}`,
     "is:pr",
     "is:merged",
     `merged:>${twoDaysAgo.toISOString()}`,
     "sort:updated",
   ].join("+");
-  const pullRequestUrl = `https://api.github.com/search/issues?q=${pullRequestsQuery}&per_page=10`;
+  const pullRequestUrl = `https://api.github.com/search/issues?q=${pullRequestsQuery}&per_page=${count}`;
   const pullRequestsData = (await got(pullRequestUrl).json()) as {
     items: any[];
   };
+  const prDataRepo = pullRequestsData.items.map((item) => ({
+    ...item,
+    repo: { name: repo, url: `https://github.com/${repo}` },
+  }));
+  return prDataRepo;
+}
 
+async function fetchRecentContributions() {
+  const nbContribPerRepo = 5;
+  const contentRecentContribs = await fetchGitHubPRs(
+    "mdn/content",
+    nbContribPerRepo
+  );
+  const translatedRecentContribs = await fetchGitHubPRs(
+    "mdn/translated-content",
+    nbContribPerRepo
+  );
+  const pullRequestsData = contentRecentContribs
+    .concat(translatedRecentContribs)
+    .sort((a, b) => a.updated_at - b.updated_at);
```

Comment:
Maybe we can make this more elegant:

```suggestion
  const repos = ["mdn/content", "mdn/translated-content"];
  const countPerRepo = 5;
  const pullRequests = (
    await Promise.all(
      repos.map(async (repo) => await fetchGitHubPRs(repo, countPerRepo))
    )
  ).flat();
  const pullRequestsData = pullRequests.sort((a, b) =>
    a.updated_at < b.updated_at ? 1 : -1
  );
```

Reply:
far more elegant indeed 🙇‍♂️